po/nl.po

```diff
@@ -192,44 +191,39 @@ msgstr "Houd jezelf gezond."
 
 #: data/com.github.elfenware.badger.appdata.xml.in:30
 msgid "Update: Screenshot"
-msgstr ""
+msgstr "Bijgewerkt: schermafdruk"
 
 #: data/com.github.elfenware.badger.appdata.xml.in:37
 msgid "Add: 🎉️ Global switch to turn off all reminders 🎉️"
-msgstr ""
+msgstr "Toegevoegd: 🎉️ optie om alle herinneringen uit te schakelen 🎉️"
 
 #: data/com.github.elfenware.badger.appdata.xml.in:44
-#, fuzzy
 msgid "Add: Turkish translation (@libreajans)"
-msgstr "Toegevoegd: Deense vertaling (@siigdev)"
+msgstr "Toegevoegd: Turkse vertaling (@libreajeans)"
```

Comment:
```suggestion
msgstr "Toegevoegd: Turkse vertaling (@libreajans)"
```

Reply:
You're right, I misread his name! Next time I'll use copy-and-paste. Sorry 'bout that, @libreajans ! And thank you for making me aware of it, @dar5hak :)